### DISCUSSION TITLE
No theme appearing

### DISCUSSION THREAD

**Discussion by bstummer** [community/community#22050] [Other Feature Feedback, Questions, & Ideas]
<p>The theme at my gh-pages site is not appearing.<br>
Repo: <a href="https://github.com/Vaschex/PartyService" class="inline-onebox" rel="noopener nofollow ugc">GitHub - Vaschex/PartyService</a></p>
<p>The support from MkDocs said that it is a GitHub specific problem: <a href="https://github.com/mkdocs/mkdocs/issues/2659" class="inline-onebox" rel="noopener nofollow ugc">MkDocs doesn't deploy · Issue #2659 · mkdocs/mkdocs · GitHub</a></p>

**Comment by jfullstackdev** [comment]
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/3X/9/5/95ff247ef946ae6e2895dbff9901fb3fe70ddcce.png" data-download-href="/uploads/short-url/loVMzPMnQCJxrTtCx2cFevCAogK.png?dl=1" title="image" rel="noopener nofollow ugc"><img src="https://user-images.githubusercontent.com/47092464/181068477-8a54345e-f2fd-4490-9e95-566f1bafc36c.png" alt="image" data-base62-sha1="loVMzPMnQCJxrTtCx2cFevCAogK" width="567" height="500" srcset="https://aws1.discourse-cdn.com/github/optimized/3X/9/5/95ff247ef946ae6e2895dbff9901fb3fe70ddcce_2_567x500.png, https://aws1.discourse-cdn.com/github/original/3X/9/5/95ff247ef946ae6e2895dbff9901fb3fe70ddcce.png 1.5x, https://aws1.discourse-cdn.com/github/original/3X/9/5/95ff247ef946ae6e2895dbff9901fb3fe70ddcce.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/3X/9/5/95ff247ef946ae6e2895dbff9901fb3fe70ddcce_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">image</span><span class="informations">838×738 63.5 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div>
<p>upon checking your site, you have <code>gh-pages</code> and in <code>main</code> you have <code>docs</code> folder so make sure you are not confused, they can all deploy your site,</p>
<p>but your GitHub Pages is working, your only problem is the theme, and upon checking your <code>index.html</code>,</p>
<p><code>&lt;link rel="stylesheet" href="assets/stylesheets/main.a57b2b03.min.css"&gt;</code></p>
<p>this is the most common problem, you can try the absolute link, or put <code>/</code> before your relative links, for example <code>/assets/ ... </code></p>
<p>wait for 5-10 minutes to see if it is working or not, because it will take time,</p>

**Comment by bstummer** [comment]
<p>That’s not the problem, since MkDocs works for others too.</p>

**Comment by jfullstackdev** [comment]
<p>okay I see, I did not notice that,</p>
<p>it is also a static site generator, but GitHub Pages has its own, Jekyll, it’s actually powered by Jekyll, so it’s best to use the default one,</p>
<p>that’s actually one common problem with using other external apps, because if you are not keen, you might not realize that there is the problem with the app when used with another one,</p>

**Comment by bstummer** [comment]
<aside class="quote no-group" data-username="jdevstatic" data-post="5" data-topic="210717">
<div class="title">
<div class="quote-controls"></div>
 jdevstatic:</div>
<blockquote>
<p>if you are not keen, you might not realize that there is the problem with the app when used with another one</p>
</blockquote>
</aside>
<p>What do you mean by that?</p>

**Comment by jfullstackdev** [comment]
<p>that it is not working with GitHub Pages, from the support of your static site generator, they are telling that the problem is from GitHub, not from them, so you really need to put effort in order to find what’s actually causing that issue,</p>

**Comment by jfullstackdev** [comment]
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/3X/0/f/0f1334b30ae17bcf261fc4a036aac2d548685296.png" data-download-href="/uploads/short-url/29mjfeI4MwFo1pt3j44qYbngTd4.png?dl=1" title="image" rel="noopener nofollow ugc"><img src="https://user-images.githubusercontent.com/47092464/181068483-86c7e77c-fe97-4f88-a6c7-6b2c2adaff7e.png" alt="image" data-base62-sha1="29mjfeI4MwFo1pt3j44qYbngTd4" width="690" height="275" srcset="https://aws1.discourse-cdn.com/github/optimized/3X/0/f/0f1334b30ae17bcf261fc4a036aac2d548685296_2_690x275.png, https://aws1.discourse-cdn.com/github/optimized/3X/0/f/0f1334b30ae17bcf261fc4a036aac2d548685296_2_1035x412.png 1.5x, https://aws1.discourse-cdn.com/github/original/3X/0/f/0f1334b30ae17bcf261fc4a036aac2d548685296.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/3X/0/f/0f1334b30ae17bcf261fc4a036aac2d548685296_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">image</span><span class="informations">1329×531 39.8 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div>
<p>yup, this is the same thing that I’m referring to, I mentioned you have <code>docs</code> in <code>main</code>, and you have <code>gh-pages</code> also,</p>
<p>my suggestion is try to deploy it again, say in <code>gh-pages</code> of a new repo, do not put <code>docs</code> in main, and make sure, you select <code>gh-pages</code> the one responsible to deploy your site,</p>
<p>if it is still causing the issue, I think, the problem is really the URL, put some time to check that, relative URLs are tricky, that’s the most common problem encountered here when deploying a page through GitHub Pages,</p>

**Comment by bstummer** [comment]
<aside class="quote no-group" data-username="jdevstatic" data-post="8" data-topic="210717">
<div class="title">
<div class="quote-controls"></div>
 jdevstatic:</div>
<blockquote>
<p>do not put <code>docs</code> in main</p>
</blockquote>
</aside>
<p>Why not? Other MkDocs users have it there as well.</p>
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/3X/7/8/78fb5c0579d5fab9430d4d26f2d347ab23944349.png" data-download-href="/uploads/short-url/hgfPyHYVcga7g4OZY6doGJB33Sh.png?dl=1" title="Screenshot_20211107-164038~2" rel="noopener nofollow ugc"><img src="https://user-images.githubusercontent.com/52933850/181068484-0d7840a7-30d7-4afa-8e91-b623cdc3e9bd.png" alt="Screenshot_20211107-164038~2" data-base62-sha1="hgfPyHYVcga7g4OZY6doGJB33Sh" width="690" height="495" srcset="https://aws1.discourse-cdn.com/github/optimized/3X/7/8/78fb5c0579d5fab9430d4d26f2d347ab23944349_2_690x495.png, https://aws1.discourse-cdn.com/github/optimized/3X/7/8/78fb5c0579d5fab9430d4d26f2d347ab23944349_2_1035x742.png 1.5x, https://aws1.discourse-cdn.com/github/original/3X/7/8/78fb5c0579d5fab9430d4d26f2d347ab23944349.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/3X/7/8/78fb5c0579d5fab9430d4d26f2d347ab23944349_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">Screenshot_20211107-164038~2</span><span class="informations">1048×753 65.8 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div><p></p>
<aside class="quote no-group" data-username="jdevstatic" data-post="8" data-topic="210717">
<div class="title">
<div class="quote-controls"></div>
 jdevstatic:</div>
<blockquote>
<p>make sure, you select <code>gh-pages</code> the one responsible to deploy your site</p>
</blockquote>
</aside>
<p>How do I do that?</p>

**Comment by jfullstackdev** [comment]
<p>yup, that’s what I’m talking to, let me check first, good you provided many details,</p>

**Comment by bstummer** [comment]
<p>This is an example of a working site: <a href="https://github.com/nanoblox/core" class="inline-onebox" rel="noopener nofollow ugc">GitHub - nanoblox/core</a></p>

**Comment by jfullstackdev** [comment, marked as answer]
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/3X/2/f/2f086ad38aeb795da6adfd137d5fc6ffe7f3d96c.png" data-download-href="/uploads/short-url/6I4tsb66X8LKDjOyQRJ22UIatFq.png?dl=1" title="image" rel="noopener nofollow ugc"><img src="https://user-images.githubusercontent.com/47092464/181068489-33f75fe4-4be9-4678-a553-6ee8869f4ad5.png" alt="image" data-base62-sha1="6I4tsb66X8LKDjOyQRJ22UIatFq" width="678" height="499" srcset="https://aws1.discourse-cdn.com/github/optimized/3X/2/f/2f086ad38aeb795da6adfd137d5fc6ffe7f3d96c_2_678x499.png, https://aws1.discourse-cdn.com/github/optimized/3X/2/f/2f086ad38aeb795da6adfd137d5fc6ffe7f3d96c_2_1017x748.png 1.5x, https://aws1.discourse-cdn.com/github/original/3X/2/f/2f086ad38aeb795da6adfd137d5fc6ffe7f3d96c.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/3X/2/f/2f086ad38aeb795da6adfd137d5fc6ffe7f3d96c_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">image</span><span class="informations">1077×794 41.5 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div>
<p>see, that’s the confusion,</p>
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/3X/8/3/8390d3600dcf2cf0f904868813496277a30c20b8.png" data-download-href="/uploads/short-url/iLSPQse4mF7j6rzww4fEEz0np1K.png?dl=1" title="image" rel="noopener nofollow ugc"><img src="https://user-images.githubusercontent.com/47092464/181068490-51ff03a8-0c25-4359-b8b2-44b6d4685812.png" alt="image" data-base62-sha1="iLSPQse4mF7j6rzww4fEEz0np1K" width="690" height="388" srcset="https://aws1.discourse-cdn.com/github/optimized/3X/8/3/8390d3600dcf2cf0f904868813496277a30c20b8_2_690x388.png, https://aws1.discourse-cdn.com/github/optimized/3X/8/3/8390d3600dcf2cf0f904868813496277a30c20b8_2_1035x582.png 1.5x, https://aws1.discourse-cdn.com/github/optimized/3X/8/3/8390d3600dcf2cf0f904868813496277a30c20b8_2_1380x776.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/3X/8/3/8390d3600dcf2cf0f904868813496277a30c20b8_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">image</span><span class="informations">1920×1080 219 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div>
<p>the <code>index.md</code> is the one being deployed and you don’t have there your background theme, it’s on <code>gh-pages</code> and GitHub Pages cannot see that,</p>
<p>so now, what you want to do is to transfer that in <code>docs</code>, transfer all your project files there, and do not change the setting on GitHub Pages,</p>
<p>now, this one is really tricky, so better create a new repo, host it in <code>docs</code> folder of <code>main</code> and put all of your project files there,</p>

**Comment by bstummer** [comment]
<p>Ahh it works now. I just changed this setting:</p>
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/3X/e/0/e0a424ad0ba1fe761f8abfd7bfed65bab6abae1a.png" data-download-href="/uploads/short-url/w3gCYJZmB7oVbba4gP01xY739KW.png?dl=1" title="Screenshot_20211107-165821~2" rel="noopener nofollow ugc"><img src="https://user-images.githubusercontent.com/52933850/181068493-2ec4adf4-487f-4349-abf6-151627d1ebd6.png" alt="Screenshot_20211107-165821~2" data-base62-sha1="w3gCYJZmB7oVbba4gP01xY739KW" width="690" height="228" srcset="https://aws1.discourse-cdn.com/github/optimized/3X/e/0/e0a424ad0ba1fe761f8abfd7bfed65bab6abae1a_2_690x228.png, https://aws1.discourse-cdn.com/github/optimized/3X/e/0/e0a424ad0ba1fe761f8abfd7bfed65bab6abae1a_2_1035x342.png 1.5x, https://aws1.discourse-cdn.com/github/original/3X/e/0/e0a424ad0ba1fe761f8abfd7bfed65bab6abae1a.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/3X/e/0/e0a424ad0ba1fe761f8abfd7bfed65bab6abae1a_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">Screenshot_20211107-165821~2</span><span class="informations">1080×358 28.4 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div>
<p>I thought that it has to be the markdown files, I’m dumb :joy::man_facepalming:<br>
Anyways, thanks for your help!</p>

**Comment by jfullstackdev** [comment]
<p>okay no problem, welcome,</p>
